lib/assessment.rb

```diff
@@ -43,6 +59,13 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    #I don't really understand the ARGV[0], just took from read-file.rb demo yesterday, and I don't know when to use it/if I need it here.
```

Comment:
ARGV is a list of arguments passed in when a program is run from the console. You'll already have the filename you need (passed in as an argument to the method) so reading from the console is unnecessary.